composer.json

```diff
@@ -3,6 +3,16 @@
     "type": "sylius-plugin",
     "description": "Sylius Plugin for order creation in Admin panel",
     "license": "MIT",
+    "authors": [
+        {
+            "name": "Mateusz Zalewski",
+            "homepage": "http://mpzalewski.com.pl"
+        },
+        {
+            "name": "Grzegorz Sadowski",
+            "email": "sadowskigp@gmail.com"
+        }
```

Comment:
Usually, we are adding a `Sylius Team` as well with generic email https://github.com/Sylius/MVM/blob/master/composer.json#L20-L23